app/src/main/java/org/hackillinois/android/view/MentorFragment.kt

```diff
@@ -1,5 +1,42 @@
 package org.hackillinois.android.view
 
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
 import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import org.hackillinois.android.R
+import org.hackillinois.android.model.mentor.MentorModel
 
-class MentorFragment : Fragment()
\ No newline at end of file
+class MentorFragment : Fragment() {
+
+    private lateinit var mentorList: List<MentorModel>
+
+    private lateinit var mentorRecycler: RecyclerView
+
+    private lateinit var myAdapter: MentorAdapter
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val view = inflater.inflate(R.layout.fragment_mentor, container, false)
+
+        mentorRecycler = view.findViewById(R.id.mentor_recycler)
+
+        val mentor1 = MentorModel("Raghav Saini", 0, "Siebel")
```

Comment:
The project service is working in the backend, so we can replace this data with actual data from the API now. Slack me if you need help with that!